www/caddy/src/opnsense/scripts/OPNsense/Caddy/caddy_certs.php

```diff
@@ -28,13 +27,8 @@
  *    POSSIBILITY OF SUCH DAMAGE.
  */
 
-require_once("config.inc");
-
 use OPNsense\Core\Config;
 
-$configObj = Config::getInstance()->object();
-$temp_dir = '/var/db/caddy/data/caddy/certificates/temp/';
-
 function extract_and_save_certificates($configObj, $temp_dir)
```

Comment:
Why is this a function? I don't see any other reference to it and it would prevent unneccesary dependencies due to split up scripts on the backend if this were just executed directly here.

Reply:
I tried to fix this php style error:

```
src/git/opnsense-plugins/www/caddy/src/opnsense/scripts/OPNsense/Caddy/caddy_certs.php
A file should declare new symbols (classes, functions, constants, etc.) and cause no other side effects, or it should execute logic with side effects, but should not do both. The first symbol is defined on line 38 and the first side effect is on line 31.
```

If its not necessary to fix I can roll back to the original script and add a style exception to it.

Reply:
Well, in this case the fix would be to remove the function and just execute the logic directly when the script is called:

> or it should execute logic with side effects, but should not do both

Reply:
Thanks for the feedback, I will improve this.